plugins/net_plugin/net_plugin.cpp

```diff
@@ -3971,6 +3981,7 @@ namespace eosio {
            "   _port  \tremote port number of peer\n\n"
            "   _lip   \tlocal IP address connected to peer\n\n"
            "   _lport \tlocal port number connected to peer\n\n")
+         ( "p2p-always-resync", bpo::value<bool>()->default_value(false), "Always attempt to resync even when peer sends blocks earlier than this node's last irreversible block")
```

Comment:
I think we should not add this new option and just make this the behavior.

Reply:
Option removed.

Reply:
PR name/description needs to be updated.

Reply:
Updated.